Dockerfile

```diff
@@ -33,8 +33,9 @@ RUN \
         python3-setuptools \
     && pip3 install -U pip setuptools packaging \
     && pip3 install -U psycopg2-binary \
```

Comment:
do we still need `psycopg2` ?

Reply:
Yeah, it's used by the diff tool that helps generate migrations.

Reply:
isn't it risky to have both psycopg2 and psycopg3 ?

Reply:
I don't see why? The modules are all very clearly psycopg or psycopg2. But I'll take a look at pgmigra to see if there have been any updates, it connects through sqlalchemy, so it's possible it could use psycopg3 now.

Reply:
I could always just run migra from it's own docker container too to keep it entirely separate.